api_tests/schema_responses/views/test_schema_response_detail.py

```diff
@@ -495,6 +495,13 @@ def test_PATCH_updates_revision_justification(self, app, schema_response, payloa
         schema_response.refresh_from_db()
         assert schema_response.revision_justification == 'why not?'
 
+    def test_PATCH_empty_revision_justification_fails(self, app, schema_response, payload, admin_user):
```

Comment:
`test_PATCH. . ._fails`?

Reply:
ahh forgot to rename it when I reverted the changes